### DISCUSSION TITLE
Do I need a Pro version?

### DISCUSSION THREAD

**Discussion by 1066258184** [sandboxie-plus/Sandboxie#2662] [General]
必须要专业版才可以吗

**Comment by okrc** [comment]
什么要专业版，你能详细描述你的问题吗？
What needs a professional version and can you describe your problem in detail?

**Reply by 1066258184** [reply]
沙盒

**Reply by okrc** [reply]
https://github.com/sandboxie-plus/Sandboxie/blob/master/CONTRIBUTING.md

无需为默认沙盒付费，你也可以多次试用其他类型沙盒5分钟。如果你想无限制的使用一些高级特性，你需要赞助者凭据。
There is no need to pay for the default sandbox, and you can also try other types of sandboxes for 5 minutes multiple times. If you want unlimited access to some advanced features, you need sponsor credentials.

**Comment by love-code-yeyixiao** [comment]
俗话说，“入乡随俗”，在Github,你需要一个英语翻译器或者一个英语四六级。
As the saying goes, “When in Rome, do as the Romans do.” At Github, you need an English translator or a cet-4 or CET-6.

Sandboxie不存在专业版一说，这是免费增值软件，你只需要证书来解锁功能，而不是重新安装一个专门版本。
There is no pro version of Sandboxie, you just need a certificate to unlock the functionality, not to reinstall a pro version.
